agent/config/config_unix_test.go

```diff
@@ -68,6 +68,7 @@ func TestConfigDefault(t *testing.T) {
 	assert.False(t, cfg.SharedVolumeMatchFullConfig.Enabled(), "Default SharedVolumeMatchFullConfig set incorrectly")
 	assert.Equal(t, defaultCgroupCPUPeriod, cfg.CgroupCPUPeriod, "CFS cpu period set incorrectly")
 	assert.Equal(t, DefaultImagePullTimeout, cfg.ImagePullTimeout, "Default ImagePullTimeout set incorrectly")
+	assert.False(t, cfg.ContainerPullInParallel.Enabled(), "Default ContainerPullInParallel set incorrectly")
```

Comment:
please update `agent/config/config_test.go` as well

Reply:
The test in `agent/config/config_tes.go` is updated in [this commit](https://github.com/aws/amazon-ecs-agent/pull/2700/commits/fba2d68cb35ed883fb1b920417c02caa77a43981#diff-98880b87ac7d759f2be9fd85c2c67fea301867244e4dcdd314be185c93ef7f7bR201). Thank you!